articles/how-to-build-astatic-site-with-gatsbyjs/index.md

```diff
@@ -0,0 +1,272 @@
+
+This is not an issue when building static sites. Due to cloud service companies like [Amazon](https://aws.amazon.com/), [Google](https://cloud.google.com/), [Netlify](https://www.netlify.com/), [Heruko](https://www.heroku.com/), and [Gaysby Cloud](https://www.gatsbyjs.com/cloud/) etc. Having worked on making software development easier, faster, and more scalable for developers.
+
+WHen compared to static sites, dynamic sites are slow to load and can cause high bounce rate on a page by users, now that’s where GatsbyJs comes into play; it is a powerful and easy static site generator.
+
+### What is GatsbyJs?
+>GatsbyJs can be defined as a static site generator that uses [React.js](https://reactjs.org/) (for the client-side) and [GraphQL](https://graphql.org/) (to access data) to build a reliable and faster website.
+
+**Advantages**
+- It's [faster](https://www.gatsbyjs.com/contributing/how-to-pitch-gatsby/) to create a static site with GatsbyJs.
+- It's easy to set up.
+- It's scalable.
+- It has a great [community of developers](https://www.gatsbyjs.com/contributing/community/).
+
+### Prerequisites
+To get started with GatsbyJs, you will need some background knowledge on the following:
+- [React.js](https://reactjs.org/) (the basics, components, etc.)
+- [Nodejs](https://nodejs.org/en/) (the basics, [npm](https://www.npmjs.com/), etc)
+- [GraphQL](https://graphql.org/) (the basics and advanced)
+
+However there’s no problem if you don’t have any knowledge on these technologies, this article will help you get started with GatsbyJs, so stick around and check the links out later if you are interested in advancing your knowledge.
+
+### Installations
+First of all, you'll need to install [Node.js](https://nodejs.org/en/) on your computer.
+Go to the [official Node.js website](https://nodejs.org/en/) to download the Node.js version for your operating system.
+
+After installing it, open up your terminal and type in:
+
+```bash
+Users/yourname: node –version
+```
+
+and
+
+```bash
+Users/yourname: npm –version
+```
+
+These commands will give you the version of Node.js and node package manager respectively.
+
+Next, in your terminal type in:
+
+```bash
+Users/yourname: npm install –global gatsby-cli
+```
+
+You just installed Gatsby and its command line interface globally in your computer, this is the first step to interact with Gatsby using the command line interface.
+
+In order to confirm you have Gatsby installed, type in:
+
+```bash
+Users/yourname: gatsby –version
+```
+
+### Creating a new GatsbyJs site
+Now let's create a static site using GatsbyJs.
+
+Open up your terminal in your code editor, if you are using [VS Code](https://code.visualstudio.com/), it comes in with a terminal in it. You will be using the GatsbyJs site template from GitHub to create your site from scratch.
+
+Create a **my-first-gatsby-site** folder, and then in your terminal type in
```

Comment:
terminal**,**

Reply:
what do you mean by this @louisefindlay23

Reply:
@Jethro-magaji I believe she was sayin there should be a comma after terminal

Reply:
Okay, but on which line number because there are many terminals there?

Reply:
> Okay, but on which line number because there are many terminals there?

Lets worry about this (,) towards the end - there are other issues we could address. 

This one is super minor 👍

Reply:
Resolved